src/clearComplete.test.js

```diff
@@ -0,0 +1,20 @@
+/** ****************************************
+ * Project: js-todo-list
+ * File: clearComplete.test.js
+ * Created: 3/9/23
+ * Author: Abdullah Al Mamun <mamun1214@gmail.com>
+ ****************************************** */
+import Apps from './modules/Apps';
+import Todo from './modules/Todo';
+
+const apps = new Apps();
+
+describe('Clear Complete', () => {
+  apps.saveItems(new Todo('Jest Test', false, 1));
+  apps.saveItems(new Todo('Jest Test 1', true, 2));
+  apps.saveItems(new Todo('Jest Test 3', true, 3));
+  apps.clearComplete();
+  it('Clear Complete Item', () => {
+    expect(apps.getItems().length).toBe(1);
+  });
+});
```

Comment:
Nice work creating the tests for the clear completed function. However, as part of the project requirements, you are to use mock for all functions that used localStorage, DOM or external APIs. Since the clear completed function uses the localStorage (the `saveItems` function on lines 13 - 15 affects the local storage) and the DOM manipulation, kindly create a mock for the testing. :pray: